javav2/example_code/cloudfront/src/main/java/com/example/cloudfront/CreateFunction.java

```diff
@@ -0,0 +1,90 @@
+//snippet-sourcedescription:[CreateFunction.java demonstrates how to create a CloudFront function.]
+//snippet-keyword:[AWS SDK for Java v2]
+//snippet-keyword:[Code Sample]
+//snippet-keyword:[Amazon CloudFront]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[02/20/2021]
+//snippet-sourceauthor:[scmacdon AWS]
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+
+package com.example.cloudfront;
+
+import software.amazon.awssdk.core.SdkBytes;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.cloudfront.CloudFrontClient;
+import software.amazon.awssdk.services.cloudfront.model.FunctionConfig;
+import software.amazon.awssdk.services.cloudfront.model.FunctionRuntime;
+import software.amazon.awssdk.services.cloudfront.model.CreateFunctionRequest;
+import software.amazon.awssdk.services.cloudfront.model.CreateFunctionResponse;
+import software.amazon.awssdk.services.cloudfront.model.CloudFrontException;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+
+
+/**
+ * To run this Java V2 code example, ensure that you have setup your development environment, including your credentials.
+ *
+ * For information, see this documentation topic:
+ *
+ * https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
+ */
+public class CreateFunction {
+
+    public static void main(String[] args) {
+
+        final String USAGE = "\n" +
+                "Usage:\n" +
+                "    <functionName> \n\n" +
+                "Where:\n" +
+                "    functionName - the name of the function to delete. \n";
```

Comment:
Not "the name of the function to delete", but to create

Reply:
fixed